openpgp.d.ts

```diff
@@ -27,10 +27,10 @@ export class Key {
   public keyPacket: PublicKeyPacket | SecretKeyPacket;
   constructor(packetlist: PacketList<AnyPacket>);
   public armor(): string;
-  public decrypt(passphrase: string | string[], keyId?: Keyid): Promise<boolean>;
-  public encrypt(passphrase: string | string[]): Promise<void>;
-  public getExpirationTime(capability?: 'encrypt' | 'encrypt_sign' | 'sign' | null, keyId?: Keyid | null, userId?: UserId | null): Promise<Date | typeof Infinity | null>; // Returns null if `capabilities` is passed and the key does not have the specified capabilities or is revoked or invalid.
-  public getKeyIds(): Keyid[];
+  public decrypt(passphrase: string | string[], keyId?: KeyId): Promise<void>; // throws on err
```

Comment:
Bit nitpicky but bytes aren't that expensive anymore nowadays 🙃 
```suggestion
  public decrypt(passphrase: string | string[], keyId?: KeyId): Promise<void>; // throws on error
```
(Also elsewhere in the file)